documentation/Windows Event Log (EVT) format.asciidoc

```diff
@@ -291,8 +298,16 @@ path specification of the event message file.
 ....
 
 [NOTE]
-This value can contain multiple filenames separated by a semi colon (;)
-character and that the name of the event message files is case insensitive.
+If set, this value must container one or more filenames separated by a semicolon (;)
+character. The names of the files are case insensitive. Values that are a directory
+or an invalid file (see below) are ignored. Filenames may contain case-insensitive
+environment variables that must be expanded.
+
+[NOTE]
+Each program that displays event message strings is responsible for reading the
+registry value, handling any environment variables within the value, and calling
+LoadLibrary on each file. Programs other than the Event Viewer may treat
+environment variables differently.
```

Comment:
"Programs other than the Event Viewer may treat environment variables differently."

what are you trying to explain with this?

Reply:
Below this the doc calls out %SystemRoot% and %WinDir%, and points to where to find these values in the registry. Logging software can use any environment variables it wants in these paths though, and the paths can vary based on the logged-in user. While uncommon, this does mean that tools that try to parse the file format manually or even just using ReadEventLog() may show different messages than Event Viewer. It's a major problem with the whole "each program has to locate and render the messages itself" design of the EVENTLOGRECORD events, and I thought it would be useful to explicitly call that out in case it saves somebody time trying to debug why their messages sometimes don't match Event Viewer's.

Reply:
It is even more complicated than that ;)

there is double indirection, see [Parameter expansion](https://github.com/libyal/libevt/blob/master/documentation/Windows%20Event%20Log%20(EVT)%20format.asciidoc#parameter-expansion)

Reply:
I tried to rephrase this to be more clear that the messages may not match what Event Viewer shows, but I'm not sure if I actually made it better or worse.

Reply:
Thx, I'll go over the changes (likely later this week) before merge, and make tweaks if necessary.